PolyEngine/RenderingDevice/OpenGL/Src/Pipeline/EnvCapture.cpp

```diff
@@ -258,12 +256,12 @@ void EnvCapture::CaptureSpecularPrefilteredMap()
 
 	Matrix ViewFromModel[] =
```

Comment:
Why is it repeated three times? Create a method or static global constant

Reply:
Done